packages/rocket-auth-aws-infrastructure/src/lambdas/challenge-verify.ts

```diff
@@ -0,0 +1,11 @@
+import { CognitoUserPoolEvent } from 'aws-lambda'
+
+export const handler = async (event: CognitoUserPoolEvent): Promise<CognitoUserPoolEvent> => {
+  const expectedAnswer = event.request.privateChallengeParameters?.secretLoginCode ?? ''
+  if (event.request.challengeAnswer === expectedAnswer) {
+    event.response.answerCorrect = true
+  } else {
+    event.response.answerCorrect = false
+  }
```

Comment:
Why not
```suggestion
  const { challengeAnswer } = event.request
  event.response.answerCorrect = challengeAnswer === expectedAnswer
```

Reply:
Fixed [here](https://github.com/boostercloud/booster/pull/556/commits/dfc0c4014bcdfd27086220b92a91a30fe6acd1af)